src/others/components/TimelineSlider.tsx

```diff
@@ -0,0 +1,49 @@
+import { Box, Grid, Slider, Typography } from "@mui/material";
+import React from "react";
+import moment from "moment";
+
+const dateFormat = 'DD.MM'
+
+export function TimelineSlider() {
+    let todayDate = moment()
+    let weekBeforeDate = moment().subtract(7, 'days')
```

Comment:
Is it possible not to hardcode the time duration?
Otherwise:
 - we'll have to hardcode it in multiple places (on the backend as well)
 - right now we do not have data for 7 days, so some values would lead to an empty screen. the way jsfiddle does it right now is: goes through the data and collects all the unique date and then creates a slider based on that info. would that be possible to do here too?

Reply:
You totally right. I refactored it with one assumption.
To this component we will pass a string array of possible dates which is sorted ascending, distinct and strings are in format DD-MM-YYYY. LMK if this is ok.